### ISSUE TITLE
The detection performance on TLESS dataset is very poor. What's wrong?

### ISSUE THREAD

**Issue by AlbertLZG** [DLR-RM/AugmentedAutoencoder#45]
Hi,
    I want to reproduce results in paper. So I trained a retinanet following the instruction:
    1. Generate a training dataset by running 'python detection_utils/generate_sixd_train.py'
    2. Train a retinanet according to ' [https://github.com/fizyr/keras-retinanet]( https://github.com/fizyr/keras-retinanet)'
    However, I only get a mean ap 18% on TLESS dataset at the threshold IOU=0.5
    So, What could be the problem? I would be grateful if you could provide detailed configuration of training retinanet.

**Comment by MartinSmeyer** [comment]
Hmm, difficult to tell. How is the performance on the training set? Did you change from training to test mode (batch norm)? Did you fine-tune from a checkpoint?

**Comment by MartinSmeyer** [comment]
Sorry, currently I don't have time to write a step-by-step manual to train a detector. Please refer to the explanations in the paper (latest version) and [https://github.com/fizyr/keras-retinanet](url)

Closing for now.
